lib/main_ltp_loader.pm

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 SUSE LLC
```

Comment:
Please use copyright without years in new files. I think this is mentioned in the contribution file as well